README.md

```diff
@@ -9,7 +9,8 @@ About
 Requirements
 ------------
 
-* Django 1.6
+* Python 2.7
+* Django 1.8
```

Comment:
Hmmmmm, I suppose we'll add it here and move the 1.7 test out of the "allow_failures" section in Travis. If anything breaks the compatibility, we can simply remove the test envs and stop listing 1.7 as compatible.